packages/react-core/src/components/DataToolbar/DataToolbarItem.tsx

```diff
@@ -18,10 +18,9 @@ export enum DataToolbarItemVariant {
 export interface DataToolbarItemProps extends React.HTMLProps<HTMLDivElement> {
   /** Classes applied to root element of the data toolbar item */
   className?: string;
-  /** A type modifier which modifies spacing specifically depending on the type of item */
+  /** TODO: Support 'separator' element as a <Divider component="div" variant="horizontal" />. A type modifier which modifies spacing specifically depending on the type of item */
```

Comment:
There does not appear to be a replacement for the Datatoolbar separator variant. Is that happening in a separate PR?

Reply:
Yep! This PR is just aiming to fix the build and move in the right direction for future work this sprint.

Another major thing left unimplemented is Tab/Nav horizontal scrolling.